assets/src/edit-story/components/library/panes/shared/chipGroup/index.js

```diff
@@ -176,9 +180,6 @@ const ChipGroup = ({ items, selectedItemId, selectItem, deselectItem }) => {
                     active={selected}
                     aria-selected={selected}
                     onClick={() => handleClick(selected, id)}
-                    // The first or selected category will be in focus for roving
-                    // (arrow-based) navigation initially.
-                    tabIndex={i === 0 || selected ? 0 : -1}
```

Comment:
This will not work correctly if fully removing the `tabIndex` -- `useRovingTabIndex` changes the tab indexes when using the arrow keys, setting `0` to the button that was last focused and `-1` to others.

Now things like this can happen since by default every button has `tabIndex` `0` but when using arrow keys to go through these, the ones that are already visited, get `-1` as the tab index instead (using only tab here after using arrow keys previously):
![tabindex](https://user-images.githubusercontent.com/3294597/113545226-114bd380-95ea-11eb-88e8-96a5ee191ed0.gif)

Probably just setting `0` to the first button and `-1` to others could work OK. Or setting `0` only to `selected`.

Reply:
I'll add this back in!